src/khoj/interface/web/assets/markdown-it.min.js

```diff
@@ -3227,6 +3227,16 @@
         result += (token.nesting === -1 ? "</" : "<") + token.tag;
     // Encode attributes, e.g. `<img src="foo"`
         result += this.renderAttrs(token);
+
```

Comment:
1. What is this token.tag code doing? Seems like it has to do with styling the github profile picture from the issues? Maybe we can add a comment to say what this code is about?

2. It'd be great if we can keep this code outside of `markdown-it.min.js` as that's an external js library and upgrading it later will be problematic if we modify it. If that's not easy/possible we should add a bold comment or something warning that this is custom code we've added to `markdown-it` so hopefully someone notices when we upgrade `markdown-it` next

Reply:
Fair point. Yes, this is for adding a specific class if this is an img tag element with a src that indicates an avatar in the URL. This allows us to have that tiny profile picture in the summary comment thread. Otherwise, it blows up in full resolution and looks unpleasant.